README.md

```diff
@@ -188,25 +188,36 @@ function paintCar(car) {
 ```
 **[⬆ back to top](#table-of-contents)**
 
-### Short-circuiting is cleaner than conditionals
+### Use default arguments instead of short circuiting or conditionals
 
 **Bad:**
 ```javascript
 function createMicrobrewery(name) {
-  var breweryName;
+  let breweryName;
   if (name) {
     breweryName = name;
   } else {
     breweryName = 'Hipster Brew Co.';
   }
+  ...
 }
 ```
 
-**Good**:
+**Better:**
```

Comment:
Thanks for more contributions!

I agree with you in principle. It does seem to me that people are going to read this non-linearly. I do give advice in a linear way of increasing cleanliness -- see the Concurrency section. If we set the precedent here, that would need to be fixed.

As for your solution, this is looking almost perfect. I don't want to change the format of this project though, so let's avoid having a Better clause. Can we just have Good/Bad?